app/models/profile.rb

```diff
@@ -106,6 +106,8 @@ def clone_to(account: nil, host: nil)
     new_profile = in_account(account)
     if new_profile.nil?
       (new_profile = dup).update!(account: account, hosts: [host])
```

Comment:
When the profile is the *first* profile of its kind in an account, we set the host properly up here. But in case the profile already existed in the account, nothing sets it.